contracts/chain-adapters/Ethereum_Adapter.sol

```diff
@@ -9,14 +9,32 @@ import "@uma/core/contracts/common/implementation/Lockable.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
+/**
+ * @notice Contract containing logic to send messages from L1 to Ethereum SpokePool.
```

Comment:
I would add a note that this adapter is used when the hub and spoke are on the same chain (L1 and Ethereum are the same thing, but by the way it's written here, that's not clear).

Reply:
We should consider renaming this contract then because the name "Ethereum" suggests it should only be deployed on Ethereum. WDYT?

Reply:
added note:

```
 * @notice This contract should always be deployed on the same chain as the HubPool, as it acts as a pass-through
 * contract between HubPool and SpokePool on the same chain. Its named "Ethereum_Adapter" because a core assumption
 * is that the HubPool will be deployed on Ethereum, so this adapter will be used to communicate between HubPool
 * and the Ethereum_SpokePool.
```